vsphere/changelog.d/17087.added

```diff
@@ -0,0 +1 @@
+[PLINT-323] Collecting a new vSphere metric: cpu.usage.vcpus.avg
```

Comment:
```suggestion
Collect a new vSphere metric: cpu.usage.vcpus.avg
```

Changelogs are mainly for customers and they don't have access to our internal board

Reply:
Committed the change.